src/date-picker/date-picker.scss

```diff
@@ -4,16 +4,23 @@
 @import '../icon/index';
 @import '../menu/index';
 
+$iui-date-picker-cell-width: 40px;
+$iui-date-picker-cell-height: 36px;
```

Comment:
Nice, so basically as if `$iui-baseline * 3` where baseline is equal to 12px.  I see what you did there. 😉

Reply:
Just set the stage..
Looks pretty good, doesnt it?